src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -441,6 +447,12 @@
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
     <Field Name="Method" Type="MethodSymbol"/>
   </Node>
+
+  <!-- Represents the GUID that is the current module's MVID. -->
+  <Node Name="BoundModuleVersionId" Base="BoundExpression">
```

Comment:
This node kind is used wherever we need to access `PrivateImplementationDetails.ModuleVersionId` because it has no symbol?